packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
+    <labels>
+        <fullName>quantic_ShowingResultsOf</fullName>
+        <value>Result {{0}} of {{1}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: Result 0-10 of 123,456</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowingResultsOf_plural</fullName>
+        <value>Results {{0}} of {{1}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: Results 0-10 of 123,456</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowingResultsOf_zero</fullName>
+        <value>Results {{0}} of {{1}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: Results 0-0 of 123,456</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowingResultsOfWithQuery</fullName>
+        <value>Result {{0}} of {{1}} for {{2}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: Result 0-10 of 123,456 for potato</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowingResultsOfWithQuery_plural</fullName>
+        <value>Results {{0}} of {{1}} for {{2}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: Results 0-10 of 123,456 for potato</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowingResultsOfWithQuery_zero</fullName>
+        <value>Results {{0}} of {{1}} for {{2}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: Results 0-0 of 123,456 for potato</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_InSeconds</fullName>
+        <value>in {{0}} second</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: in 1 second</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_InSeconds_plural</fullName>
+        <value>in {{0}} seconds</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>ex: in 2.3 seconds</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_InSeconds_zero</fullName>
```

Comment:
Why do we need a different one for 0 seconds? Also is 0 seconds even possible?

Reply:
No but @lbergeron pointed out that the rules for plurality with zero differ from one language to the next. so now it looks creates a `_zero` variant. We likely won't need this label ever but the util will return a label name that doesn't exist otherwise. It's not a perfect solution...